src/System.Net.HttpListener/src/System/Net/Windows/HttpListener.Windows.cs

```diff
@@ -451,6 +451,7 @@ private unsafe void CloseRequestQueueHandle()
             {
                 if (NetEventSource.IsEnabled) NetEventSource.Info($"Dispose ThreadPoolBoundHandle: {_requestQueueBoundHandle}");
                 _requestQueueBoundHandle?.Dispose();
+                _requestQueueBoundHandle = null;
```

Comment:
It doesn't look like .NET Framework nulls it out (or if it does I'm missing it).  Why doesn't the bug repro there (or does it)?

Reply:
I did not try Framework. I can give it try tomorrow.

Reply:
> I did not try Framework. I can give it try tomorrow.

Thanks.  I'm asking because the cited issue states that this is not failing on netfx, and as I don't see this being nulled out there, it seems something else may have changed to cause this.

Reply:
I could not find requestQueueBoundHandle in Framework, just requestQueueHandle. It seems like the core implementation is somewhat different but I'm not too familiar with it. 

It seems like setting this to null after disposing was reasonable but I also verified that everything seems to work as expected if we do not Dispose _requestQueueBoundHandle. Maybe that is safer fix as we don't want to impact pending IO.